bin/diagnostic.js

```diff
@@ -1 +1,15 @@
 'use strict'
+
+const ask = require('../lib/ask.js')
+
+let answer = 'not empty'
+
+while (answer !== '') {
+  answer = ask("Guess my secret? ")
```

Comment:
Make sure that you are actually providing a number for the user to guess, in addition to making sure their guess is between 1-10